tasks/package.js

```diff
@@ -50,6 +50,15 @@ module.exports = function(grunt) {
     tasks.push('clean:packages');
     tasks.push('copy:package');
 
+    // Rebuild composer dependencies without dev.
+    grunt.config(['composer'], {
+      options: {
+        flags: ['no-dev'],
+        cwd: destPath
+      }
+    });
+    tasks.push('composer:install');
```

Comment:
Can anyone think of a reason someone would create a task that executes package followed by a composer operation that needs -dev?